pyintegration/deephaven2/table.py

```diff
@@ -42,11 +99,121 @@ def columns(self):
 
         return self._schema
 
-    def update(self, formulas: List[str]):
+    def to_string(self, num_rows: int = 10, cols: List[str] = []) -> str:
+        """ Returns the first few rows of a table as a pipe-delimited string.
+
+        Args:
+            num_rows (int): the number of rows at the beginning of the table
+            cols (List[str]): the list of column names
+
+        Returns:
+            object:
+
+        Raises:
+            DHError
+        """
+        try:
+            return TableTools.string(self._j_table, num_rows, *cols)
+        except Exception as e:
+            raise DHError(e, "table to_string failed") from e
+
+    #
+    # Table operation category: Select
+    #
+    def drop_columns(self, cols: List[str]) -> Table:
```

Comment:
I think we should move the "_columns" methods to the end of the file. Personally I feel like they are somewhat superficial; I think we should generally order the methods like: selects, filters, joins, sorts, bys, other.

Reply:
I mostly followed the new doc https://deephaven.io/core/docs/reference/table-operations/select/drop-columns/ on the order of groups and methods. @chipkent ?